### ISSUE TITLE
Italicized and bolded text in description

### ISSUE THREAD

**Issue by ZetaSmile** [chaoticbackup/chaoticcardcreator#4]
Users may wish to add custom keywords to card descriptions. Certain cards in the physical game introduced new keywords alongside an italicized blurb explaining the keyword's effect(s).

This feature would add the ability to tag sections of the description text as italicized or bolded (e.g. [i]text[/i]).

**Comment by DanudeSandstorm** [comment]
I think we'd be better off following [markdown (basic syntax)](https://www.markdownguide.org/cheat-sheet/)

This is going to be a tricky to implement with how the current text drawing works

**Comment by ZetaSmile** [comment]
That's true. It seems that online suggestions for ctx are to split the text into multiple sections and render them individually, but this would quickly get complex.
